### DISCUSSION TITLE
视频下载FLV格式 是否已经去除了4K选项?

### DISCUSSION THREAD

**Discussion by Lichtmann** [the1812/Bilibili-Evolved#3773] [提问 / Q&A]
在4K视频作品里下载FLV格式的没有了4K选项
![image](https://user-images.githubusercontent.com/29507184/197376975-cac740b1-20d4-4b42-93eb-1fa9fe091982.png)
![image](https://user-images.githubusercontent.com/29507184/197376989-86e5f296-4ed3-454d-88c1-02fb56b6b9e2.png)

**Comment by the1812** [comment]
可能吧, flv 要逐渐没落了

**Comment by Github-TBH** [comment]
前几天我也发现了这个问题，我还以为是浏览器的问题
